### ISSUE TITLE
Creating email channel cause always results of 2 email creation

### ISSUE THREAD

**Issue by roneybd85** [mautic/mautic#13336]
### Mautic Version

5.0.x series

### Way of installing

I installed with composer using https://github.com/mautic/recommended-project

### PHP version

8.1.27

### What browsers are you seeing the problem on?

Chrome

### What happened?

I expect upon saving email chanel , only one email channel should be created . this also happen with campaign email

### How can we reproduce this issue?

Step 1: goto Channel > email > create a new segment email . save it
step 2 : check the link of emails . with same email title there will be 2 instead of one

this happens with campaign emails too

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by escopecz** [comment]
This seems like a duplicate of https://github.com/mautic/mautic/issues/13271 and should be resolved in Mautic 5.1.0
